Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -210,6 +210,27 @@ private function getSelectedDebitBrand($brand)
         return null;
     }
 
+    /**
+     * @param string $item
+     * @return string
+     */
+    protected function retrieveImageUrl($item): string
+    {
+        $objectManager =\Magento\Framework\App\ObjectManager::getInstance();
```

Comment:
We shouldn't use ObjectManager, dependency injection would be the best approach to use the `\Magento\Catalog\Helper\Image` class here https://devdocs.magento.com/guides/v2.4/extension-dev-guide/depend-inj.html#constructor-injection